formatting/parser.go

```diff
@@ -182,10 +184,9 @@ func parseString(s string) ([]Item, error) {
 					tree = append(tree, item)
 
 					item = Item{
-						Text: "",
-						Decorators: []decorators.Decorator{
-							decorator,
-						},
+						Text:       "",
```

Comment:
raw motd: `§b§n§oLand of the §b§n§o§lGronkleberries §8GROOOB`

before:
![image](https://github.com/mcstatus-io/mcutil/assets/37940332/69168caf-07dd-4b8c-a94c-cc0606dcfd24)
after:
![image](https://github.com/mcstatus-io/mcutil/assets/37940332/2ce597b8-8e59-43a3-be1b-87c1c6ded6b3)